examples/using-redux/gatsby-ssr.js

```diff
@@ -1,2 +1 @@
-import wrapWithProvider from "./wrap-with-provider"
-export const wrapRootElement = wrapWithProvider
+export { wrapRootElement } from "./gatsby-browser"
```

Comment:
I wouldn't put this in example. It works here because `gatsby-browser` doesn't make use of any browser specific APIs, but in more complex examples this can cause `window is not defined` errors when generating .html files.

Does this fix anything or just shuffle code around?

Reply:
@pieh I came across your solution for the problem described here https://github.com/gatsbyjs/gatsby/issues/11494#issuecomment-460058623. Based on your comment, I used a simplified version of it in my project. I thought of updating the example in the docs since it worked for me and it was simpler.

Reply:
We can still not repeat ourselves without importing `gatsby-browser` file in `gatsby-ssr` - similar to how it is right now. This separation ensure we won't get errors when user will want to use `window` object inside `gatsby-browser`

In example adding
`window.title = "something"` to gatsby-browser file would result in broken build with this change

Reply:
We can still make it nicer to read by using 
```diff
-import wrapWithProvider from "./wrap-with-provider"
-export const wrapRootElement = wrapWithProvider
+export { wrapWithProvider as wrapRootElement } from "./wrap-with-provider"
```
(if we support that `export ... from` with export as (not sure if we do at this point)

Reply:
@pieh fair enough. I personally never assume there is a `window` because we run JS nowadays in many different places. I saw the Redux example (that's what my PR refactores). I think at this point it's probably not worth merging my change if you are not 100% sure about it. You know the Gatsby code and users better than me. I can cancel the PR if you want so we don't have to spend more time on this little thing :)

Reply:
I think if that syntax works - this would be valuable to update our example - so if you have a chance to test it out and just change current import and export to single export from (with import rename) and it works - I feel like it would be nicer example (and you will have merged pull request - it always feels nice to have that!)

Reply:
We don't "mock" `window` in SSR intentionally because it might lead to subtle bugs and it's better to explicitly fail build if `window` is being accessed in `ssr` environment (IMO) than not fail and cause deploy to have potential uncaught bugs

Reply:
@alexlbr Hi! Do you have a chance to test this?

Reply:
Hi @LekoArts! sure, I'll do it this week :)